java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -153,7 +153,8 @@ open class StorageService : ResurrectorService() {
             intent.getParcelableExtra<ParcelableStoreOptions?>(EXTRA_OPTIONS)
         ) { "No StoreOptions found in Intent" }
 
-        val options = parcelableOptions.actual
+        val options =
+            parcelableOptions.actual.copy(coroutineContext = coroutineContext)
```

Comment:
I'm surprised these are serializable/parcelable

Reply:
No they are not: this line basically specifies a coroutine context on the receiver (service) side.

Reply:
But it's creating the object via intent.getParcelableExtra(), so how does the parcelableOptions.actual field get populated? Won't this result in the invocation of ParcelableStoreOptions.CREATOR.createFromParcel() which creates a new, default StoreOptions data class instance as the .actual field? It seems to me that parcelableOptions.actual will always be a Dispatchers.Default because that's the default in the StoreOptions ctor.

Reply:
I'm actually passing in the coroutineContext of the StorageService to this options.